### DISCUSSION TITLE
Multiple G-code commands

### DISCUSSION THREAD

**Discussion by slwise** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2848] [Q&A]
In the "Custom G-code Commands" section of the config.ini file, is it possible to execute more than one G-code command per custom menu item or is it limited to one. Thanks for the help.

**Comment by radek8** [comment]
You can use more than one command, but a maximum of 75 characters.
Each G-code must be terminated by a \n character

**Reply by slwise** [reply]
Thank you so much!
